components/Participation/index.jsx

```diff
@@ -67,6 +67,7 @@ export const ParticipationSection = () => {
                   carreiras acima, mas ainda não tem experiência (seu perfil
                   Junior será analisado pelo Linkedin),
                   <b>
+                    {" "}
```

Comment:
Isso aqui é meio bizarro, a solução é essa mesma?

Reply:
Verifiquei que o prettier está removendo os espaços no final ou início de cada tag, podemos usar &nbsp; para forçar um espaçamento.